app/api/auth/users/userData/nameTag/route.ts

```diff
@@ -0,0 +1,68 @@
+import { NameTagContent } from "@/components/NameTagForm";
+import startDB from "@/lib/db";
+import UserModel from "@/models/userModel";
+import { NextRequest, NextResponse } from "next/server";
+
+/**
+ * GET request to fetch the user's nametag
+ */
+
+type FetchUserNametagResponse = NextResponse<{
+  success?: boolean;
+  error?: string;
+  nameTag?: NameTagContent;
+}>;
+
+export const GET = async (req: NextRequest): Promise<FetchUserNametagResponse> => {
+    const userEmail = req.nextUrl.searchParams.get("userEmail");
+
+    if (userEmail) {
```

Comment:
I often find these cases easier to read when putting the "early return" cases first, as then you can get rid of some of the nesting and make the success case a bit easier to find.

e.g., changing things like:
```
if (something) {
    if (something else) {
        A;
    } else {
      fail(B);
   }
  fail(C);
} 
```
to
```
if (!something) {
  fail(C);
}
if (!something else) {
  fail(B);
}
A;
```

Reply:
Good idea! Updated this file: https://github.com/aimpowered/ZoomCompanion/pull/60/commits/96baa90ff4faef723f15c5c0a1384973d0bcde17

Reply:
For `if` cases that end in a `return` (or `throw`), you can also remove the `else` branch and associated indentation.

Reply:
> For `if` cases that end in a `return` (or `throw`), you can also remove the `else` branch and associated indentation.

https://github.com/aimpowered/ZoomCompanion/pull/60/commits/33580d4cc57b88def58b1cb7a30f8b229b45511b